src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -224,9 +224,8 @@ void CommandRequestImpl::onTimeOut() {
                                             function_id(),
                                             correlation_id(),
                                             mobile_api::Result::GENERIC_ERROR);
-
+  (*response)[strings::msg_params][strings::info] = info;
```

Comment:
@pold500 Yes, this way the "info" parameter will always be present in this response which may be an error behavior for some cases.